bzt/resources/base-config.yml

```diff
@@ -156,6 +156,11 @@ services:
     - conn-all
 
 modules: # TODO: cleanup as much as possible from defaults
+  blazemeter:
+    dedicated-ips: false
```

Comment:
Allow me to clarify: we don't need these defaults as defaults. However, without these defaults in place, these fields are simply removed from any yml that includes them, and never make it to the server. The yml received by the server doesn't include them. So this is simply the only way we found to force taurus to keep these particular fields in the yml.